aeron-driver/src/main/c/aeron_driver_conductor.c

```diff
@@ -3050,6 +3071,50 @@ void aeron_driver_conductor_on_linger_buffer(void *clientd, void *item)
     }
 }
 
+void aeron_driver_conductor_on_re_resolve_endpoint(void *clientd, void *item)
+{
+    aeron_driver_conductor_t *conductor = clientd;
+    aeron_command_re_resolve_t *cmd = item;
+    struct sockaddr_storage resolved_addr;
+    memset(&resolved_addr, 0, sizeof(resolved_addr));
+
+    if (aeron_name_resolver_resolve_host_and_port(
+        &conductor->name_resolver, cmd->endpoint_name, AERON_UDP_CHANNEL_ENDPOINT_KEY, true, &resolved_addr) < 0)
+    {
+        // TODO: What is the best error handling here, error counter for re-resolution failures???
```

Comment:
Since it is a re-resolve it feels like a counter to me.

Reply:
@tmontgomery and I were thinking that the distinct error log based on name would be useful as it would quickly highlight a bad name.

Reply:
need to know the name that failed, so, yes. Error log feels more useful.

Reply:
I've now added the error logging to the Java side.

Reply:
Given many different names could be used then maybe we might want to keep a resolution report similar to the loss report to aid debugging?

Reply:
that's an interesting idea. We could. I'm very interesting in getting some feedback from users and seeing what evolves.

Reply:
I'm inclined to leave it in the error log.  A resolution report could be useful, but it would be in addition to the error log.  Mostly because it reduces the number of places an operator needs to look when a system is not working.